apps/users/handlers.py

```diff
@@ -1,425 +1,156 @@
+from apps.authorization.services import get_hashed_password
 from apps.common.exceptions import BackendException
-from apps.common.utils import get_hashed_password
-from apps.common.db import database
-from apps.users.models import User, user_list_view, Student, OneTimeToken, UserFaculty, students_list_view
-from apps.users.schemas import UserOut, TokenPayload, UsersListViewOut, CreateUserOut, CreateUserIn, DeleteUserIn, \
-    RegistrationOut, RegistrationIn, CreateStudentOut, CreateStudentIn, StudentsListOut, UserIn, DeleteStudentIn, \
-    StudentCheckExistanceOut, StudentCheckExistanceIn
-from settings import Settings
-
-from random import randint
-from typing import List, Union
-from datetime import datetime, timedelta
-from jose import jwt
-
-import hashlib
-import os
-
-from sqlalchemy import select, insert, delete, update
-from translitua import translit
-from fastapi import Depends, APIRouter, HTTPException, status as http_status
-from fastapi.security import OAuth2PasswordBearer
-from pydantic import ValidationError
-from apps.common.schemas import JSENDOutSchema, JSENDFailOutSchema
-
-
-users_router = APIRouter()
-
-
-reuseable_oauth = OAuth2PasswordBearer(
-    tokenUrl="/login",
-    scheme_name="JWT"
-)  # TODO spelling mistake 'reusable'
-
-
-async def get_current_user(token: str = Depends(reuseable_oauth)) -> UserOut:
-    try:
-        payload = jwt.decode(
-            token, Settings.JWT_SECRET_KEY, algorithms=[Settings.JWT_ALGORITHM]
-        )
-        token_data = TokenPayload(**payload)
-
-        if datetime.fromtimestamp(token_data.exp) < datetime.now():
-            raise HTTPException(
-                status_code=http_status.HTTP_401_UNAUTHORIZED,
-                detail="Token data has expired",
-                headers={"WWW-Authenticate": "Bearer"}
+from apps.common.utils import get_login, get_student_attr, get_token_data, get_login_full_name,\
```

Comment:
delete \ and put items in list. see example at line 8-9

Reply:
ok